uPy/html.py

```diff
@@ -0,0 +1,191 @@
+# Simple HTML parser for PMT to use
```

Comment:
Remove your name and add the PMT header from another file then update to what this file needs in that header.  Everything below that is fine, except the structure of return value, delete that you are reiterating